src/openloco/map/tilemgr.cpp

```diff
@@ -189,23 +190,23 @@ namespace openloco::map::tilemgr
     {
         if ((input::getMapSelectionFlags() & 1) != 0)
         {
-            for (uint16_t x = _mapSelectionAX; x <= _mapSelectionBX; x += 32)
+            for (coord_t x = _mapSelectionAX; x <= _mapSelectionBX; x += 32)
             {
-                for (uint16_t y = _mapSelectionAY; y <= _mapSelectionBY; y += 32)
+                for (coord_t y = _mapSelectionAY; y <= _mapSelectionBY; y += 32)
                 {
-                    map_invalidate_tile_full(x, y);
+                    auto pos = map::map_pos(x, y);
```

Comment:
You don't need to construct this variable and then use it just do `map_invalidate_tile_full({x, y});`